pkgs/build-support/setup-hooks/make-wrapper.sh

```diff
@@ -32,26 +32,20 @@ makeWrapper() {
     for ((n = 2; n < ${#params[*]}; n += 1)); do
         p="${params[$n]}"
 
-        if test "$p" = "--set"; then
+        if [[ "$p" == "--set" ]]; then
```

Comment:
A previous PR that did something similar (#19061) ran up against the OSX system version of bash being too old to support the case statement.